composer.json

```diff
@@ -1,14 +1,117 @@
 {
     "name": "zendframework/skeleton-application",
-    "description": "Skeleton Application for ZF2",
+    "description": "Skeleton Application for Zend Framework",
     "license": "BSD-3-Clause",
     "keywords": [
         "framework",
         "zf2"
     ],
     "homepage": "http://framework.zend.com/",
+    "minimum-stability": "dev",
```

Comment:
chicken-egg problem. We can keep it as is, and it will install stable packages when detected. I setup the constraints to prefer stable packages as well (`^3.0 || ^3.0.0-dev@dev`), so leaving it will work for now. Once zend-mvc and zend-test are tagged, we can remove the line. (And that will happen before we prepare this repository for release.)